pandas/plotting/_matplotlib/core.py

```diff
@@ -369,6 +408,28 @@ def result(self):
             else:
                 return self.axes[0]
 
+    def _transform_grouped_data(self, data: ABCDataFrame) -> ABCDataFrame:
+        """
+        Internal function to transform grouped DataFrame object to a normal
```

Comment:
yeah, i was thinking of adding an example other than describing it, but realize it is quite different to do it in `core.py` file, because this does not seem to be callable to use in example in docstring. Any idea how to do this? @datapythonista

Reply:
The example will be useful while reading the code. Not sure what's your point here, but an example in docstrings will let us understand what the functions are expected to do.